extensions/wikia/EditPageLayout/js/editor/AceEditor.js

```diff
@@ -0,0 +1,199 @@
+define('wikia.editpage.ace.editor', ['wikia.ace.editor'], function(ace){
+	'use strict';
+
+	var theme = 'solarized_light',
+		isEditorWide = false,
+		inputAttr = {
+			name: 'wpTextbox1',
+			id: 'wpTextbox1'
+		},
+		$editPage = $('#EditPage'),
+		$editPageToolbar = $('#EditPageToolbar'),
+		wideClassName = 'editpage-sourcewidemode-on',
+		narrowClassName = 'editpage-sourcewidemode-off',
+		beforeEditorInit = function(){
```

Comment:
Looks like this could be defined using standard notation:

function beforeEditorInit () {...}